docs/concepts/what-is-a-workflow-execution.md

```diff
@@ -2,57 +2,122 @@
 id: what-is-a-workflow-execution
 title: What is a Workflow Execution?
 sidebar_label: Workflow Execution
-description: A Workflow Execution is a Reentrant Process; that is, a resumable, recoverable, and reactive process.
+description: A Temporal Workflow Execution is a durable, scalable, reliable, and reactive function execution. It is a Temporal Application’s main unit of execution.
 tags:
   - explanation
 ---
 
-A Workflow Execution is a Reentrant Process; that is, a resumable, recoverable, and reactive process:
+A Temporal Workflow Execution is a durable, reliable, and scalable function execution.
+It is a Temporal Application’s main unit of execution.
 
-- Resumable: Ability of a process to continue execution after execution was suspended on an await-able.
-- Recoverable: Ability of a process to continue execution after execution was suspended on a failure.
-- Reactive: Ability of a process to react to external events.
+- [How to spawn a Workflow Execution in Go](/docs/go/how-to-spawn-a-workflow-execution-in-go)
+- [How to spawn a Workflow Execution in Java](/docs/java/how-to-spawn-a-workflow-execution-in-java)
+
+Each Temporal Workflow Execution has exclusive access to its local state, executes concurrently to all other Workflow Executions, and communicates with other Workflow Executions and the environment via message passing.
+
+A Temporal Application can consist of millions to billions of Workflow Executions.
+Workflow Executions are lightweight components.
+A Workflow Execution consumes few compute resources; in fact, if a Workflow Execution is suspended, such as when it is in a waiting state, the Workflow Execution consumes no compute resources at all.
+
+**Durability**
+
+Durability is responsiveness over a period time.
+A Workflow Execution is durable because it executes a Temporal Workflow Definition, also called a Temporal Workflow Function, your application code, effectively once and to completion—whether your code executes for seconds or years.
+Durability is realized through the execution semantics of a Workflow Execution, which are carried out via Temporal's Event loop.
```

Comment:
This sounds confusing to me. 

I think durability is about persisting workflow execution state in the presence of various software and hardware failures.

Reply:
Per @dtornow - sounds like this is track Product is taking:
Durability is the absence of an imposed time limit.
Reliability is responsiveness in the presence of failure.
Scalability is responsiveness in the presence of load.